src/C-DEngine/C-DEngine.csproj

```diff
@@ -24,7 +24,7 @@
 		<!-- Workaround for placing each DLL into a separate directory when using msbuild's GenerateProjectSpecificOutputFolder -->
 		<ProjectName>$(AssemblyName).$(TargetFramework)</ProjectName>
 		<!-- Workaround for placing the .nupkg into the build's output folder when using msbuild's GenerateProjectSpecificOutputFolder -->
-		<PackageOutputPath>$(OutDir)</PackageOutputPath>
+		<PackageOutputPath>$(SolutionDir)Nugets</PackageOutputPath>
```

Comment:
This breaks nuget publishing (see .github/workflows/dotnetcore.yml).
An approach would be to change the location of the global bin directory to be under the SolutionDir. This can be done in Directory.Build.props, line 32 - 37: just remove the first  ..\ for both ==cdeUpdater and !=cdeUpdater conditions.

Reply:
As long as it is under (SolutionDir) I can work with it. The problem is that other Solutions that target docker need to have the nugets in a "docker accessible" path under the SolutionDir or docker cannot find the nugets that are not published to nuget.org

Reply:
Should be resolved now